docs/policies/vulnerability_management_policy.md

```diff
@@ -38,14 +38,18 @@ The digital service teams and portfolios are responsible for ensuring they mitig
 
 ### Responsibile Accountable Consulted Informed (RACI) matrix
```

Comment:
Sorry Sam, I should've clocked this earlier but while I'm here now - can we update 'Responsible'?